### ISSUE TITLE
New Regression: N64 Overlays Analog Left are no longer function and remappable.

### ISSUE THREAD

**Issue by Yabuato** [libretro/RetroArch#6700]
Note: I used @blackman91 'd issue template.

Description
Using newest nightly(May.2.2018) RetroArch 1.7.2  in the controls menu the touchscreen overlays Analog left are no longer function and remappable, they were remappable in previous RetroArch stable(68).

Expected behavior
1.Analog Left is function when toggle digital dpad to analog left on overlays. 2.Analog Left is remappable when open quick menu>controls>remap analog left and the analog left will get remapped.

Actual behavior
1.Analog Left is not function when toggle digital dpad to analog left on overlays.
 2.Analog Left is not remappable when open quick menu>controls>remap analog left and analog left can't be remapped.

Steps to reproduce the bug
1.Use N64 Touchscreen overlay
2.Open quick menu
3.Select controls
4.Remap Analog Left
Bisect Results
After the new nightly RetroArch update that changed the input mapping.

Version/Commit
Version: 1.7.2_Git
Commit:468b41f

Environment information
Android 4.4.2

**Comment by blackman91** [comment]
I think so, just tested it and it seems that with the recent overlay fix (great job btw) just the analog in any touchscreen overlay no longer works in the parallel n64 core and mupen cores to move even if you enable the left analog in controls menu. The analog of the touchscreen overlay seems to work on other cores though. Also external gamepad analog is working, so it just seems to be an issue with the oberlay analog in n64 cores.

**Comment by twinaphex** [comment]
@fr500 Seems there could still be an issue with the analog stick as far as remapping goes.

**Comment by Yabuato** [comment]
@hizzlekizzle 

Yes,analog left does work (works but not remappable) in previous stable build.It doesn't work (and not remappable) in this nightly build.

Rest of the buttons remap issue have been fixed(thanks for this).

**Comment by fr500** [comment]
I don't provide remapping for analogs in overlays, and I can't reproduce this with mupen.

**Comment by fr500** [comment]
Should be fixed once the fix is merged

**Comment by Yabuato** [comment]
That's fast it's perfect now,really appreciate your work.
